### DISCUSSION TITLE
Export ResultSet and numbers format

### DISCUSSION THREAD

**Discussion by nanny07** [dbeaver/dbeaver#363] [Q&A] [answered]
Hi,

is there a way to export a result set with the native database format? This is a big problem when there are columns with float/money/decimal format: if I use the export tools, it will format the number with my current culture's format. For example, "1200.01" is formatted with "1.200,01" but this will be a problem if I need to use the value with others program

The only trick I found is to use advance copy and check "database native" format, but I can't use for big tables

**Comment by serge-rider** [comment, marked as answer]
You can change numbers formatting in preferences or right in export wizard (Formatting option on the 3rd page of wizard).
In numbers format turn "Use Grouping" off.

**Comment by nanny07** [comment]
Thanks you, it just what I was looking for
